ddl/sequence.go

```diff
@@ -125,7 +125,7 @@ func handleSequenceOptions(SeqOptions []*ast.SequenceOption, sequenceInfo *model
 				sequenceInfo.Start = mathutil.MinInt64(sequenceInfo.MaxValue, model.DefaultNegativeSequenceStartValue)
 			}
 			if !minSetFlag {
-				sequenceInfo.MaxValue = model.DefaultNegativeSequenceMinValue
+				sequenceInfo.MinValue = model.DefaultNegativeSequenceMinValue
```

Comment:
bug fix?

Reply:
yes, A previous mistake, should open another PR to do this?

Reply:
If the previous PR wasn't cherry-picked, I think just fix it in this PR is OK.

Reply:
ok